docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -721,6 +761,89 @@ def uses_db_connection():
 
 If a resource you want to initialize has dependencies on other resources, those can be included in the dictionary passed to <PyObject object="build_resources"/>. For more in-depth usage, check out the [Initializing Resources Outside of Execution](/concepts/resources#initializing-resources-outside-of-execution) section.
 
+### Monitoring asset partitions with sensors
+
+#### Updating a weekly asset partition when upstream daily partitions are materialized
```

Comment:
This is probably a nit, but if there's a way to shorten this heading (and the other one in this section) I think it's worth a shot. It's getting squashed in the page nav and is pretty long for a heading in general.